package.json

```diff
@@ -28,8 +28,8 @@
     "commit": "git-cz"
   },
   "peerDependencies": {
-    "react": "^16.6",
-    "react-dom": "^16.6",
+    "react": "^16.8",
```

Comment:
@ej9x Is it ok that we are limiting developers to use only latest react?
I think it shouldn't be so strict.

Reply:
@vorobeez And what is your solution?

Reply:
Just thinking out loud.